contracts/simple_dex/lib.rs

```diff
@@ -102,23 +102,31 @@ mod simple_dex {
     }
 
     #[ink(event)]
-    pub struct SwapPairAdded {
+    pub struct Withdrawn {
+        caller: AccountId,
```

Comment:
maybe caller also should be marked as topic?

Reply:
What I'm thinking rn is that this whole tx (deposit) makes 0 sense - anyone can circumvent it by just transferring PSP22 to the dex address, in fact this is how we'll be adding liquidity.

This is a leftover form when the DEX was more complex and included tracking LP, but it makes no sense in this simplified design. 

 I'm removing it, along with the event